### ISSUE TITLE
Specific parameter setting when training on activityNet v1.2

### ISSUE THREAD

**Issue by zronly** [sujoyp/wtalc-pytorch#15]
Hi, I trained your pytorch code  on activityNet v1.2 dataset. But I can only get the results as follows.
tIou@0.1= 47,  tIou@0.2= 44,  tIoU@0.3=40, tIou@0.4=37,  tIou@0.1= 33....it is much lower than what you posted. I think there's something wrong with the parameters. Can you share the specfic paramaters when training on activitNetv1.2 ? I would appreciate it very much if I could receive your reply.

**Comment by HumamAlwassel** [comment]
@zronly how were you able to get the results you cited? I'm trying to do the same from my end, but I get mAP @ IoU=0.5 that's extremely low (less than 1%). The classification mAP reaches +90%.

I'm running the ActivityNet1.2 experiment with the following settings: 
`python main.py --dataset-name ActivityNet1.2 --num-class 100 --model-name AN1.2_reprduce
`
I edited the smooth() function in detectionMAP.py to include the savgol_filter() call. I have played around with the threshold, but the detection mAP is still extremely low.

**Comment by sujoyp** [comment]
Please refer to this: https://github.com/sujoyp/wtalc-pytorch/issues/7
Also need to use savgol_filter as @HumamAlwassel mentioned.

**Comment by zronly** [comment]
> Please refer to this: #7
> Also need to use savgol_filter as @HumamAlwassel mentioned.
Yes, i have used the same setting as what you said, but i find the CASLOSS is not work when training the activitynet1.2 , because when  the casloss goes down, the accuracy  also goes down. Is this caused by batch size? or learning rate?

**Comment by simpler66** [comment]
@zronly i have test the results on ＡctivitＮet1.2, and get the same results as yours mAP＠0.5=33.5, if you have got the results posted in paper, please let me know.
I run experiments on ActivityNet1.2 with the following settings:
 python main.py --dataset-name ActivityNet1.2 --num-class 100
I do not change any other parameters in options.py provided by author used for training thumos14.
@sujoyp I will be really appreciate if you could offer the option.py used for ActivityNet

**Comment by zronly** [comment]
> @zronly i have test the results on ＡctivitＮet1.2, and get the same results as yours mAP＠0.5=33.5, if you have got the results posted in paper, please let me know.
> I run experiments on ActivityNet1.2 with the following settings:
> python main.py --dataset-name ActivityNet1.2 --num-class 100
> I do not change any other parameters in options.py provided by author used for training thumos14.
> @sujoyp I will be really appreciate if you could offer the option.py used for ActivityNet

the casloss don't work when training on ActivityNet1.2, i am not sure if it has anything to do with parameters.
